gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/CourseGradeOverridePanel.java

```diff
@@ -100,10 +101,19 @@ public void onSubmit(final AjaxRequestTarget target, final Form<?> form) {
 
 				if (StringUtils.isNotBlank(newGrade)) {
 					final Map<String, Double> schema = gbInfo.getSelectedGradingScaleBottomPercents();
-					if (!schema.containsKey(newGrade)) {
-						error(new ResourceModel("message.addcoursegradeoverride.invalid").getObject());
-						target.addChildren(form, FeedbackPanel.class);
-						return;
+					
+					if (!schema.containsKey(newGrade))
+					{
```

Comment:
Can we keep the formatting the same i.e. `if (something) {`

The brace styling with the brace on a separate line is not the preferred style!

Probably the most acceptable style is now google's checkstyle http://google.github.io/styleguide/javaguide.html